CHANGES

```diff
@@ -4,6 +4,7 @@ Pint Changelog
 0.17 (unreleased)
```

Comment:
0.17 has been released can you rebase ?

Reply:
I merged the master branch into my develop. Should do the trick